src/shared/database/entities/crypto-data.entity.ts

```diff
@@ -0,0 +1,31 @@
+import { Column, Entity } from 'typeorm';
+import { Base } from './base.entity';
+
+@Entity('cryptoData')
+export class CryptoDataEntity extends Base {
+  @Column()
```

Comment:
You don't have any relations between entities this way. You should use `@ManyToOne`, `@OneToOne`, etc.

Reply:
mongodb doesn't have any relations